tests/lib/rules/no-deprecated-destroyed-lifecycle.js

```diff
@@ -71,8 +71,8 @@ ruleTester.run('no-deprecated-destroyed-lifecycle', rule, {
       code: `
       <script>
       export default {
-        beforeUnmount:beforeDestroy,
-        unmounted:destroyed,
+        beforeUnmount,
+        unmounted,
```

Comment:
Why did you need to change this?

Reply:
sorry, that's a misoperation, updated now